pkgs/development/libraries/libgit2/default.nix

```
@@ -46,7 +46,23 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = lib.optional (!stdenv.isLinux) libiconv;
 
-  doCheck = false; # hangs. or very expensive?
+  doCheck = true;
+  checkPhase = ''
+    testArgs=(-v -xonline)
+
+    # slow
+    testArgs+=(-xclone::nonetwork::bad_urls)
+
+    # failed to set permissions on ...: Operation not permitted
+    testArgs+=(-xrepo::init::extended_1)
+    testArgs+=(-xrepo::template::extended_with_template_and_shared_mode)
+
+    ( set -x
+      ./libgit2_tests ''${testArgs[@]}
+    )
```

```suggestion
    (
      set -x
      ./libgit2_tests ''${testArgs[@]}
    )
```
I am used to shfmt google style which I am pretty sure would suggest this